homeassistant/components/binary_sensor/uptimerobot.py

```diff
@@ -0,0 +1,87 @@
+"""A component to monitor Uptime Robot monitors.
+
+For more details about this component, please refer to the documentation at
+https://www.home-assistant.io/components/binary_sensor.uptimerobot
+"""
+import logging
+import voluptuous as vol
```

Comment:
Please separate the three import groups, standard library, 3rd party and homeassistant, with a blank line.